apps/ledger-live-desktop/src/renderer/families/casper/operationDetails.jsx

```diff
@@ -0,0 +1,43 @@
+// @flow
+import React from "react";
+import { Trans } from "react-i18next";
+import {
+  OpDetailsTitle,
+  OpDetailsData,
+  OpDetailsSection,
+} from "~/renderer/drawers/OperationDetails/styledComponents";
+import Ellipsis from "~/renderer/components/Ellipsis";
+import type { Account } from "@ledgerhq/types-live";
+
+type OperationDetailsExtraProps = {
+  extra: { [key: string]: any },
+  type: string,
+  account: Account,
+};
+
+const OperationDetailsExtra = ({ extra }: OperationDetailsExtraProps) => {
```

Comment:
Same comment again about upcoming https://github.com/LedgerHQ/ledger-live/pull/4235, provided it gets merged first you'll just have to update the props here.